rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -848,6 +848,14 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	SubscribeTooltip(ui->gb_DiskCacheClearing, tooltips.settings.limit_cache_size);
 	connect(ui->enableCacheClearing, &QCheckBox::stateChanged, ui->maximumCacheSize, &QSlider::setEnabled);
 
+	// Date Time Edit Box
+	m_emu_settings->EnhanceDateTimeEdit(ui->console_time_edit, emu_settings_type::ConsoleTimeOffset, tr("dd MMM yyyy HH:mm"), true, true, 15000);
```

Comment:
I'm not really satisfied with the 15 seconds.
Let's say you keep the calendarWidget open for a while and time progresses.
You then close the popup.
-> The real time will have progressed by a couple of minutes, but the widget will show the old time for another 15 seconds in the worst case, making you believe that you just changed the time by a couple of minutes.

Reply:
Is there a time value you'd prefer?

Ideally I'd say that as soon as focus is lost from both the QDateTimeEdit widget and the QCalendar widget that it should immediately update to reflect the proper time (based on offset), but the framework seems to have times during a genuine editing event where neither of these indicates it has focus (for a few seconds).

Reply:
idk. it's your PR :)